src/fides/api/ctl/database/session.py

```diff
@@ -15,7 +16,7 @@
 sync_engine = create_engine(CONFIG.database.sync_database_uri, echo=False)
 sync_session = sessionmaker(
     sync_engine,
-    class_=Session,
+    class_=ExtendedSession,
```

Comment:
this helped my tests run, and it _seemed_ like a good idea regardless since our `ExtendedSession` just provides some nice error handling around the base ORM session - so why not use it with our `ctl` session? 

i think we have plans to consolidate our `session` objects anyway across `ops` and `ctl` code paths, so this is just a step in that direction. all tests pass so hopefully we are good?